core/chains/starknet/chain_set.go

```diff
@@ -8,20 +8,22 @@ import (
 
 	"github.com/smartcontractkit/sqlx"
 
-	"github.com/smartcontractkit/chainlink-starknet/relayer/pkg/starknet"
-	"github.com/smartcontractkit/chainlink-starknet/relayer/pkg/starknet/db"
+	starkchain "github.com/smartcontractkit/chainlink-starknet/relayer/pkg/chainlink/chain"
```

Comment:
The original plan was to put Chainlink type implementations into the `chainlink/starknet` nested package, so we'd implement a types like `starknet.Chain/Set`